modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2484,6 +2523,75 @@ private Map<String, Serializable> _getValuesFromDatabase(
 		return values;
 	}
 
+	private void _testAddObjectEntryExternalReferenceCode() throws Exception {
+		ObjectEntry objectEntry1 = _addObjectEntry(
```

Comment:
@p-albuquerque please rename it to `objectEntry` as we don't have another `objectEntry` here